rootfs/common/INSTALL_PREFIX/scripts/logpull/collect/qca.sh

```diff
@@ -31,12 +31,12 @@
 
 collect_qcawl()
 {
-    collect_cmd iwconfig
+    collect_cmd iw dev
 
     # Collect list of STA clients
     for IF in $(cat /proc/net/wireless | sed 1,2d | grep -v wifi | cut -d : -f 1); do
         collect_cmd wlanconfig $IF list sta
-        collect_cmd iwlist $IF scan last
+        collect_cmd iw dev $IF scan
```

Comment:
This has a nasty side-effect of blocking and performing a full blown scan. The "iwlist" was used with "last" argument to only report the BSS cache without performing an actual scan.

The "iw" has a similar variant, `iw wlanXXX scan dump` (note, the `dump` argument). Can you please use that instead? Also, see my other comment - it would be much better to extend the device-core, see https://github.com/plume-design/device-core/blob/master/rootfs/common/INSTALL_PREFIX/scripts/logpull/collect/linux.sh

Reply:
I was checking other opensync boxes, and it seems /proc/net/wireless is not so standard and is not available on multiple platforms -- so maybe it still makes sense to keep this part of the code in platform/qca.

Regarding the `dump` argument I agree, @quic-rjangir please update the code.

Reply:
That's an excellent point. I missed this in my earlier rundown. The /proc/net/wireless will not contain any MLO interfaces to begin with, and its no longer enabled in many upstream linux builds by now either (https://www.spinics.net/lists/linux-wireless/msg262163.html)